### DISCUSSION TITLE
How to dynamically specify plugin priority?

### DISCUSSION THREAD

**Discussion by 1798972** [Kong/kong#9212] [Help] [answered]
Now we all define the priority in the plug-in's handle.lua, like the following.
![image](https://user-images.githubusercontent.com/49218070/182786064-0816e1cd-3136-423a-b25a-9d96d51d8f04.png)
How to dynamically specify this priority when applying plug-ins, such as schema pages?

**Comment by hakancelikdev** [comment, marked as answer]
It would be a great feature to be able to dynamically set the priorities of the plugins.

**Reply by Tieske** [reply]
@jschmid1 is this scheduled for 3.0, or only Enterprise?

**Reply by jschmid1** [reply]
This is scheduled for 3.0 Enterprise only, unfortunately.
